contrib/conan/configs/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+function conan_disable_public_remotes {
+  conan remote disable bintray || return $?
+  conan remote disable conan-center || return $?
+  conan remote disable bincrafters || return $?
+  return 0
+}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [ "$(uname -s)" == "Linux" ]; then
+  conan config install "$DIR/linux" || exit $?
+else
+  conan config install "$DIR/windows" || exit $?
```

Comment:
Does this ever get used?

Reply:
Yes, I use that quite a lot from git-bash. It's not integrated yet but I want to call this from my `build_and_upload_dependencies.sh` script which also works on Windows.